docs/CHANGELOG.rst

```diff
@@ -13,6 +13,8 @@ Changed
 -----
 * Remove ``threads`` parameter from ``cuffquant`` and ``cufnorm``
   helper functions
+* Modify ``get_bam`` utility function to correctly work with 
+  ``bam-split`` processed samples
```

Comment:
Point out that this change is required to work with the latest process version - it is not a bug.